plugins/wrangler/api_token.go

```diff
@@ -0,0 +1,52 @@
+package wrangler
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIToken,
+		DocsURL:       sdk.URL("https://developers.cloudflare.com/workers/wrangler/system-environment-variables/"),
+		ManagementURL: sdk.URL("https://dash.cloudflare.com/profile/api-tokens"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.AccountID,
+				MarkdownDescription: "The account ID for the Workers related account, can be found in the Cloudflare dashboard, can usually be inferred by Wrangler.",
+				Secret:              true,
+				Optional:            true,
+				Composition: &schema.ValueComposition{
+					Charset: schema.Charset{
+						Uppercase: true,
+						Lowercase: true,
+						Digits:    true,
+					},
+				},
+			},
+			{
+				Name:                fieldname.Token,
+				MarkdownDescription: "The API token for your Cloudflare account, can be used for authentication for situations like CI/CD, and other automation.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Charset: schema.Charset{
+						Uppercase: true,
+						Lowercase: true,
+						Digits:    true,
+					},
```

Comment:
Don't these have a fixed length? Same for the account ID.

Reply:
It's nothing documented, So I don't wanted to add a limitation

Reply:
I'd be okay with using what we know now. Nothing bad will happen if it turns out that the length is incorrect.

Reply:
> Nothing bad will happen if it turns out that the length is incorrect.

This is probably worth documenting. @florisvdg shall we add this to https://developer.1password.com/docs/cli/shell-plugins/contribute/#credential-information-and-schema ?